ups/drp_pipe.table

```diff
@@ -14,6 +14,7 @@ setupRequired(jointcal)
 setupRequired(fgcmcal)
 setupRequired(meas_transiNet)
 setupRequired(cp_verify)
+setupRequired(ap_pipe)
```

Comment:
@TallJimbo I think this is the first time that a _pipe package depends on another _pipe package so I think you may want to jump in and declare that this is fine or not.

Reply:
Migrated the relevant tasks to `pipe_tasks`.